packages/webdriverio/src/utils/interception/webdriver.ts

```diff
@@ -12,6 +12,9 @@ export default class WebDriverInterception extends Interception {
     mockId?: string;
 
     async init () {
+        if (this.url instanceof RegExp) {
+            throw new Error('Regular Expressions as mock url are not supported')
```

Comment:
Is it possible to somehow use regex as mock ur?

Reply:
@mgrybyk yes, the patch implements it, this is the WebDriver implementation that I wanted to have for running tests agains Sauce. However I haven't been able to release this yet, so no one will be impacted by this.

Reply:
I will have to make changes to our internal driver to accept a regex from the WebDriver request (probably transformed into a string). Not sure when and if I ever get to that.